plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/assembler.page.js

```diff
@@ -16,9 +16,10 @@ export class AssemblerPage {
 			'.cys-fullscreen-iframe[style="opacity: 1;"]'
 		);
 
+		// wait for longer to support multi env testing
 		await frame
 			.getByRole( 'button', { name: 'Finish customizing' } )
-			.waitFor( { timeout: 25000 } );
+			.waitFor( { timeout: 40000 } );
```

Comment:
Wow, that's a lot of waiting time. Is it that slow in these environments?

Reply:
In my local testing, yes, but that's maybe because of my local machine 🤔

Reply:
Many times, when I ran assembler tests locally, I had to increase the time, so this should solve some local testing problems for everyone, as locally executing this kind of test can take longer than 25 seconds to load.

Reply:
I think leaving a 40s wait time in here is problematic.
If:
- it really takes that long for some environments 
and
- we really think it's valuable to run that test for those environments

then we should add some logic to increase the timeout only for those envs. Also, I would check with the dev team, maybe there's something that can be improved.

Reply:
Reverted in a new PR https://github.com/woocommerce/woocommerce/pull/50464